tests/test_plotting.py

```diff
@@ -736,46 +690,6 @@ def test_plot_basis_fourierzernike(self):
         return fig
 
 
-class TestPlotFieldLines:
```

Comment:
removed with the relevant functions. Will eventually make a better version (in another PR)